### PR TITLE
fix unstable tests

### PR DESCRIPTION
**Pull request by LavrovArtem**
/cc @miherlosev

### CODE REVIEW

**Comment by LavrovArtem** [comment]
@testcafe-build-bot \retest

**Comment by LavrovArtem** [comment]
@testcafe-build-bot \retest

File: test/client/fixtures/sandbox/backup-test.js [hunk 1]
```diff
@@ -36,26 +36,21 @@ asyncTest('"permission denied" error is raised when an iframe with a nested ifra
 
     window.QUnitGlobals.waitForIframe(iframe)
         .then(function () {
-            iframe.contentWindow.location.reload();
+            iframe.contentWindow.oldState = true;
```
Comment (miherlosev) [inline test/client/fixtures/sandbox/backup-test.js]: Let's rename to `iframe.contentWindow.testFlag`

**Comment by LavrovArtem** [comment]
@testcafe-build-bot \retest

**Pull request merged**